src/pages/workspace/distanceRates/PolicyDistanceRatesPage.tsx

```diff
@@ -243,6 +243,9 @@ function PolicyDistanceRatesPage({
     };
 
     const toggleRate = (rate: RateForList) => {
+        if (rate.isDisabledCheckbox) {
+            return;
+        }
```

Comment:
```suggestion
```
@ChavdaSachin we don't need this code bcoz `onSelectRow` only executes if the item is disabled.